### PR TITLE
Fix scanFile signature to avoid boring warning

### PR DESCRIPTION
**Pull request by PVince81**
@oparoz @icewind1991

### CODE REVIEW

**Comment by icewind1991** [comment]
:+1: makes sense

**Comment by LukasReschke** [comment]
:+1:

File: apps/files_sharing/lib/external/scanner.php [hunk 1]
```diff
@@ -48,9 +48,11 @@ public function scan($path, $recursive = self::SCAN_RECURSIVE, $reuse = -1, $loc
 	 * @param int $reuseExisting
 	 * @param int $parentId
 	 * @param array | null $cacheData existing data in the cache for the file to be scanned
+	 * @param bool $lock set to false to disable getting an additional read lock during scanning
+	 * @param array | null $cacheData existing data in the cache for the file to be scanned
```
Comment (nickvergessen) [inline apps/files_sharing/lib/external/scanner.php]: invalid doc block

Comment (PVince81) [inline apps/files_sharing/lib/external/scanner.php]: I copied this from the original signature

Comment (PVince81) [inline apps/files_sharing/lib/external/scanner.php]: argh I see it now

Comment (PVince81) [inline apps/files_sharing/lib/external/scanner.php]: https://github.com/owncloud/core/pull/17325

**Comment by oparoz** [comment]
@karlitschek https://github.com/owncloud/core/issues/17455 Backport?

**Comment by karlitschek** [comment]
Yes. please backport. Thanks :-)

**Comment by enoch85** [comment]
@oparoz Care to backport? This fix wasn't in 8.1.1.

**Comment by oparoz** [comment]
Sorry, I missed the green light...

**Pull request merged**